server/src/logic/controller/formater.js

```diff
@@ -1,113 +1,96 @@
-    entreprise_date_radiation: e.entreprise_date_radiation,
-    entreprise_siret_siege_social: e.entreprise_siret_siege_social,
-    raison_sociale: e.raison_sociale,
-    nom_commercial: e.nom_commercial,
-    siret_siege_social: e.siret_siege_social,
-    uais_formations: e.uais_formations,
-    created_at: e.created_at,
-    last_update_at: e.last_update_at,
-    entreprise_tranche_effectif_salarie: e.entreprise_tranche_effectif_salarie,
-    etablissement_siege_id: e.etablissement_siege_id,
-    matched_uai: e.matched_uai,
-    id_mna_etablissement: e._id,
-  };
+const formation = {
+  etablissement_reference: 1,
+  etablissement_reference_typeetablissement_reference: 1,
+  etablissement_formateur_siretetablissement_reference: 1,
+  etablissement_formateur_enseigneetablissement_reference: 1,
+  etablissement_formateur_uaietablissement_reference: 1,
+  etablissement_formateur_adresseetablissement_reference: 1,
+  etablissement_formateur_code_postaletablissement_reference: 1,
+  etablissement_formateur_localiteetablissement_reference: 1,
+  etablissement_formateur_complement_adresseetablissement_reference: 1,
+  etablissement_formateur_entreprise_raison_socialeetablissement_reference: 1,
+  etablissement_formateur_cedexetablissement_reference: 1,
+  etablissement_formateur_sirenetablissement_reference: 1,
+  etablissement_gestionnaire_siretetablissement_reference: 1,
+  etablissement_gestionnaire_enseigneetablissement_reference: 1,
+  etablissement_gestionnaire_uaietablissement_reference: 1,
+  etablissement_gestionnaire_adresseetablissement_reference: 1,
+  etablissement_gestionnaire_code_postaletablissement_reference: 1,
+  etablissement_gestionnaire_localiteetablissement_reference: 1,
+  etablissement_gestionnaire_complement_adresseetablissement_reference: 1,
+  etablissement_gestionnaire_cedexetablissement_reference: 1,
+  etablissement_gestionnaire_entreprise_raison_socialeetablissement_reference: 1,
+  etablissement_gestionnaire_sirenetablissement_reference: 1,
+  nom_academieetablissement_reference: 1,
+  num_academieetablissement_reference: 1,
+  code_postaletablissement_reference: 1,
+  code_commune_inseeetablissement_reference: 1,
+  num_departementetablissement_reference: 1,
+  uai_formationetablissement_reference: 1,
+  nometablissement_reference: 1,
+  intitule_longetablissement_reference: 1,
+  intitule_courtetablissement_reference: 1,
+  diplomeetablissement_reference: 1,
+  niveauetablissement_reference: 1,
+  cfdetablissement_reference: 1,
+  mef_10_codeetablissement_reference: 1,
+  rncp_codeetablissement_reference: 1,
+  rncp_intituleetablissement_reference: 1,
+  parcoursup_referenceetablissement_reference: 1,
+  parcoursup_a_chargeretablissement_reference: 1,
+  affelnet_referenceetablissement_reference: 1,
+  affelnet_a_chargeretablissement_reference: 1,
+  _idetablissement_reference: 1,
+  etablissement_formateur_idetablissement_reference: 1,
+  etablissement_gestionnaire_idetablissement_reference: 1,
```

Comment:
It looks like all the fields end with a undesired `etablissement_reference` string ;)